lib/taskana-core/src/main/java/pro/taskana/user/internal/UserMapper.java

```diff
@@ -54,6 +54,10 @@ public interface UserMapper {
   @InsertProvider(type = UserMapperSqlProvider.class, method = "insert")
   void insert(User user);
 
+  @InsertProvider(
+      type = UserMapperSqlProvider.class,
+      method = "insertGroupsOracle",
+      databaseId = "oracle")
```

Comment:
personally I don't like that we reference the databaseId through a regular string (not only here, in all sql files). 
What do you think of something like this? [opinion only] https://stackoverflow.com/a/16384334

Reply:
https://taskana.atlassian.net/browse/TSK-1999